lib/ansible/utils/module_docs_fragments/purestorage.py

```diff
@@ -21,6 +21,34 @@ class ModuleDocFragment(object):
 
     # Standard Pure Storage documentation fragment
     DOCUMENTATION = '''
+options:
+  - See seperate platform section for more details
+requirements:
+  - See seperate platform section for more details
+notes:
+  - Ansible modules are available for the following Pure Storage products: FlashArray, FlashBlade
+'''
+
+    # Documentation fragment for FlashBlade
```

Comment:
You have defined these in capital letters here, but used them as lowercase in the actual modules.  This wont work.

Reply:
@sivel I took examples of this from other files and they seem to work by using capitals in the fragment file and lowercase in the actual code. Is the best practice to use all lower or all upper case in both?

Reply:
Looks like I am wrong, seems to be standard to do it this way.